tests/test_date_picker.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.select import Select
+from selenium.webdriver.support.wait import WebDriverWait
+from utils.base_test import BaseTest
+
+BASE_URL = 'https://demoqa.com/date-picker'
+
+
+class TestToolsQADatePicker(BaseTest):
+    def setUp(self) -> None:
+        self._setUp(BASE_URL)
```

Comment:
Parent's class method should be called setUp() and you can call it like this:

super(TestToolsQADatePicker, self).setUp(URL)